eng/psakefile.ps1

```diff
@@ -397,7 +397,19 @@ task run-examples {
         )
         Assert (@(Compare-Object $bz_first_lines $bz_expected).Length -eq 0) "Expected $bz_expected found $bz_first_lines"
     }
-    
+   
+    exec -workingDirectory $pyqir.evaluator.examples_dir {
+        & $python "teleport.py" | Tee-Object -Variable teleport_output
+        $teleport_first_lines = @($teleport_output | Select-Object -first 5)
```

Comment:
What was the reason to only look at the first case, opposed to comparing all 4 cases?

Reply:
They are smoke tests to make sure the samples run without errors and generate some expected output.

Reply:
Since the code that we point people to runs for all 4 cases, I think it would make sense to also just include the 4 cases here. It's only a tiny tweak in the build script. I see that the PR is still in draft, so I'll hold off with approving - let me know when it is ready to be reviewed.

Reply:
It isn't a simple change. The last bits of the output change based on the result stream. We have an exhaustive teleport tests which verify all of this behavior. The `run-examples` task verifies that the samples run and don't generate errors. If we want to have all of the examples fully verified during the build, we should put that on the backlog as a new task.

Reply:
Alright. I realized I never updated my comment after clarifying offline with you whether we have test coverage. As said there, I'll leave the judgement call up to you, as long as we have unit tests for it somewhere.